flac-1.3.0 removed FLAC from pkg-config's includedir, so the user of flac.pc is supposed to
include the headers with FLAC/ prefix now.

Upstream-Status: Pending

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>

diff -uNr vlc-1.1.11.orig/modules/codec/flac.c vlc-1.1.11/modules/codec/flac.c
--- vlc-1.1.11.orig/modules/codec/flac.c	2014-02-23 01:29:56.337515018 +0000
+++ vlc-1.1.11/modules/codec/flac.c	2014-02-23 01:29:04.477514993 +0000
@@ -38,8 +38,8 @@
 #include <vlc_codec.h>
 #include <vlc_aout.h>
 
-#include <stream_decoder.h>
-#include <stream_encoder.h>
+#include <FLAC/stream_decoder.h>
+#include <FLAC/stream_encoder.h>
 
 #include <vlc_block_helper.h>
 #include <vlc_bits.h>
